tensorboard/plugins/interactive_inference/witwidget/notebook/jupyter/js/lib/wit.js

```diff
@@ -36,35 +37,48 @@ var WITView = widgets.DOMWidgetView.extend({
    * Loads up the WIT element.
    */
   loadAndCreateWhatIfToolElement: function() {
+    const height = parseInt(
+      this.model.attributes.layout.attributes.height, 10) - 20;
+    const iframe = document.createElement('iframe');
     const templateLocation =
-        window.__webpack_public_path__ + 'wit_jupyter.html';
+      window.__webpack_public_path__ + 'wit_jupyter.html';
+    const src = `<link rel="import" href="${templateLocation}">
```

Comment:
I expect that this will break very soon. HTML imports are deprecated and
will be removed in Chrome M75 (we’re on M74 right now). TensorBoard’s
HTML imports are compiled statically, so we don’t immediately have this
problem, but it looks like this is using HTML imports at runtime. Have
you tested this on Chrome canary?

Depending on what exactly is in `wit_jupyter.html`, you may be able to
achieve the same behavior by, within the iframe, (1) `fetch`ing that
page, (2) setting the `document.body.innerHTML` to its source, and (3)
rehydrating `<script>` elements. You can see how we do that for the
`%tensorboard` Colab widget here:

<https://github.com/tensorflow/tensorboard/blob/1925d060fbfa71fa8e26d793a94f704b7ba413e3/tensorboard/notebook.py#L314-L364>